dpc-go/dpc-api/swaggerui/dpc-api-swagger.json

```
@@ -0,0 +1,218 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "description": "As patients move throughout the healthcare system, providers often struggle to gain and maintain a complete picture of their medical history.\n        Data at the Point of Care fills in the gaps with claims data to inform providers with secure, structured patient history, past procedures, medication adherence, and more.",
+    "version": "2.0.0",
+    "title": "Data at the Point of Care",
+    "license": {
+      "name": "Public Domain",
+      "url": "https://github.com/CMSgov/dpc-app/blob/master/LICENSE.md"
+    }
+  },
+  "tags": [
+    {
+      "name": "Metadata"
+    },
+    {
+      "name": "Group"
+    },
+    {
+      "name": "Organization"
+    }
+  ],
+  "servers": [
+    {
+      "url": "https://prod.dpc.cms.gov",
```

Shall we add sandbox here too?

---

In fact, given our audience, maybe we should replace this with
```suggestion
      "url": "https://sandbox.dpc.cms.gov",
```